build_library/extra_sysexts.sh

```diff
@@ -1,5 +1,26 @@
 EXTRA_SYSEXTS=(
-  zfs:sys-fs/zfs
-  podman:app-containers/podman,net-misc/passt
-  python:dev-lang/python,dev-python/pip
+  "zfs|sys-fs/zfs"
+  "podman|app-containers/podman,net-misc/passt"
+  "python|dev-lang/python,dev-python/pip"
+  "nvidia-drivers-535|x11-drivers/nvidia-drivers:0/535|-kernel-open"
+  "nvidia-drivers-535-open|x11-drivers/nvidia-drivers:0/535|kernel-open"
+  "nvidia-drivers-550|x11-drivers/nvidia-drivers:0/550|-kernel-open"
+  "nvidia-drivers-550-open|x11-drivers/nvidia-drivers:0/550|kernel-open"
 )
+
+_get_unversioned_sysext_packages_unsorted() {
+  for sysext in "${EXTRA_SYSEXTS[@]}"; do
+    IFS="|" read -r SYSEXT_NAME PACKAGE_ATOMS USEFLAGS < <(echo "$sysext");
+
+    IFS=,
+    for atom in $PACKAGE_ATOMS
+    do
```

Comment:
For consistency with other usages:
```suggestion
    for atom in $PACKAGE_ATOMS; do
```